boards/xtensa/qemu_xtensa/qemu_xtensa.dts

```diff
@@ -0,0 +1,13 @@
+/dts-v1/;
+
+#include "sample_controller.dtsi"
+
+/ {
+	model = "qemu_xtensa";
+	compatible = "sample,controller";
```

Comment:
compatible is usually used for driver selection.  Is "sample" or "controller" really unique enough?  I know there's no "driver" for the CPU, but the principle still seems to be violated here.

"sample_controller" is an Xtensa string identifying the CPU that is implemented by qemu-xtensa.  I don't think it's reasonably decomposable into being both "sample" and "controller"